backend/src/main/java/gov/cdc/usds/simplereport/db/repository/ApiUserRepository.java

```diff
@@ -10,7 +10,8 @@
 /** Interface specification for fetching and manipulating {@link ApiUser} entities */
 public interface ApiUserRepository extends EternalSystemManagedEntityRepository<ApiUser> {
 
-  String NAME_ORDER = " order by last_name, first_name, middle_name, internal_id";
+  String NAME_ORDER =
+      " order by nameInfo.lastName, nameInfo.firstName, nameInfo.middleName, internalId";
```

Comment:
no op curiosity: was this ordering not working before? seems that `nameInfo` got added a while back and guessing that we never updated this ordering rule?

Reply:
I think this was working before, but with the Hibernate 6 upgrade I think they added [new requirements](https://docs.jboss.org/hibernate/orm/6.0/migration-guide/migration-guide.html#query-path-comparison) for paths. I can't find anything specifically for this case, but I think it's because nameInfo is embedded it requires the path to be named.

Reply:
Why all the columns but `internalId` need the row prefix?

Reply:
It's hard to pinpoint a change in their change log why this change was required, but since [nameInfo is an embedded](https://github.com/CDCgov/prime-simplereport/blob/b55e000791c45194eb797d3e2761eda778fcd94a/backend/src/main/java/gov/cdc/usds/simplereport/db/model/ApiUser.java#L23) object it should have the prefix. InternalId is a properly directly on the ApiUser object.